.github/workflows/docker-image.yml

```diff
@@ -9,17 +9,18 @@ on:
 
 env:
   IMAGE_NAME: webots
-  DEFAULT_WEBOTS_VERSION: R2022a
+  CLOUD_IMAGE_NAME: webots.cloud
+  DEFAULT_WEBOTS_VERSION: R2022b
 
 jobs:
   dockerhub-publication:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        BASE_IMAGE: ["nvidia/cudagl:11.4.2-devel-ubuntu18.04", "nvidia/cudagl:11.4.2-devel-ubuntu20.04"]
+        BASE_IMAGE: ["nvidia/cudagl:11.4.2-devel-ubuntu20.04"]
         include:
-          - BASE_IMAGE: "nvidia/cudagl:11.4.2-devel-ubuntu18.04"
-            WEBOTS_PACKAGE_PREFIX: "_ubuntu-18.04"
+          - BASE_IMAGE: "nvidia/cudagl:11.4.2-devel-ubuntu20.04"
```

Comment:
The latest version is 11.7.0, see https://hub.docker.com/r/nvidia/cuda.
We may take the opportunity of this PR to upgrade.

Reply:
I think it is still 11.4.2: https://hub.docker.com/r/nvidia/cudagl